### ISSUE TITLE
Can't set breakpoints for vue components in VS Code. 

### ISSUE THREAD

**Issue by johnlwebb3** [microsoft/vscode-js-debug#1799]
Opening this issue on GitHub is really just a last-ditch effort. I've been struggling with this for days. I've tried setting up new default projects and can't get it to work. I've researched and seen that many developers have difficulty with this. I've studied and tried various solutions all without success. apparently, many people just give up and do all their debugging in the browser dev tools. I could do that too, but I'd really rather use the VS Code IDE. 

Here are some basic facts about my situation;
1. Using Vue3 @vue/cli 5.0.8
2. Using VS Code 1.82.0
3. Using webpack@5.85.1
4. Using VS Code, I can set breakpoints in main.ts and they work.
5. Using VS Code, to set breakpoint in a vue component results in a typical red solid dot on the line of code as long as the app is not running
6. Using VS Code and launching the application causes the red solid dots to become empty white circles for breakpoints in vue components
7. Using Chrome Dev Tools - Source, **I can** locate the source file and set a breakpoint there and when it hits, VS Code breaks and everything seems to work. I want to set the breakpoint with vs code and not chrome though. **Really, I think this is the main issue I am having. I can set the breakpoints int he browser devtools but I can't set the breakpoints in VS Code.** 
8. My launch.json file;
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch Chrome",
            "request": "launch",
            "type": "chrome",
            "url": "http://localhost:8080",
            "webRoot": "${workspaceFolder}",
        },
    ]
}
```
9. my vue.config.js file:
```
let myPublicPath = process.env.VUE_APP_BASE_URL;

const { defineConfig } = require("@vue/cli-service");
module.exports = defineConfig({
  publicPath: myPublicPath,
  transpileDependencies: true,
  configureWebpack: {
    devtool: 'source-map',
  },
  devServer: {
    devMiddleware: {
      writeToDisk: true,
    },
  },
});

```

I'm prepared to work with anyone who will work with me to try and solve the issue. Thanks for any help you might be able to provide.

**Comment by connor4312** [comment]
Please collect a log file using the following instructions:

> If you're able to, add `"trace": true` to your `launch.json` and reproduce the issue. The location of the log file on your disk will be written to the Debug Console. Share that with us.
>
>⚠️ This log file will not contain source code, but will contain file paths. You can drop it into https://microsoft.github.io/vscode-pwa-analyzer/index.html to see what it contains. If you'd rather not share the log publicly, you can email it to connor@xbox.com

**Comment by johnlwebb3** [comment]
I ran it with the "trace":true in my launch.json and here is the file it produced. I see there are a lot of sourceMap entries in it so I'm hopeful we can get this working. Thanks for your help...


[vscode-debugadapter-33c8a5af.json.gz](https://github.com/microsoft/vscode-js-debug/files/12590531/vscode-debugadapter-33c8a5af.json.gz)

**Comment by connor4312** [comment]
You appear to be hitting https://github.com/vuejs/vue/issues/11023. Unfortunately the Vue team hasn't yet engaged on the issue, but please feel free to lend your support to it to increase its visibility.

**Comment by connor4312** [comment]
By the way, I notice this no longer happens in Vue 3's out of the box setup using Vite, so that is probably why the issue is stagnant. Maybe you can switch over to that to solve the issue.

**Comment by johnlwebb3** [comment]
Thank you Connor for looking at this. I spent a good many hours over several days and am glad to be moving on. I suspected VS Code had some sort of issue with the breakpoints but couldn't quit trying to make it work. 

Also, your final comment about Vite set me on a path to investigate what it would take to migrate my project from VueCLI to Vite. It took a few hours but after I got it running with Vite, the breakpoints worked perfectly. 

I wish I had posted my issue to you guys earlier! Thanks again!

John
